cmd/influxd/run/server.go

```diff
@@ -70,7 +70,8 @@ type Server struct {
 	BindAddress string
 	Listener    net.Listener
 
-	Logger *zap.Logger
+	Logger    *zap.Logger
+	MuxLogger *log.Logger
```

Comment:
Pulled this up to the top level so we can redirect it during tests and not see a bunch of "failed to connect" logs from startup.